### PR TITLE
A new task page for assigining a pod to nodes using node affinity

### PR DESCRIPTION
**Pull request by rajeshdeshpande02**
This PR is for adding new task page for assigning a pod to nodes using node affinity. It coveres both required and preferred affinities.

### CODE REVIEW

File: content/en/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+---
+
+{{% capture overview %}}
+This page shows how to assign a Kubernetes Pod to a particular node using Node Affinity in a
+Kubernetes cluster.
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md]: When using `{{< version-check >}}`, make sure to set `min-kubernetes-server-version` in the front matter metadata.

File: content/en/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+1. Verify that your chosen node has a `disktype=ssd` label:
+
+    ```shell
+    kubectl get nodes --show-labels
+    ```
+
+    The output is similar to this:
+
+    ```shell
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md]: This sample output is not a valid shell program.

File: content/en/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    NAME      STATUS    ROLES    AGE     VERSION        LABELS
+    worker0   Ready     <none>   1d      v1.13.0        ...,disktype=ssd,kubernetes.io/hostname=worker0
+    worker1   Ready     <none>   1d      v1.13.0        ...,kubernetes.io/hostname=worker1
+    worker2   Ready     <none>   1d      v1.13.0        ...,kubernetes.io/hostname=worker2
+    ```
+
+    In the preceding output, you can see that the `worker0` node has a
+    `disktype=ssd` label.
+
+## Assign pod to node using required node affinity
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md]:
```suggestion
## Create a Pod with scheduling constraints
```
?
I'm not sure about wording but I think it's good to mention scheduling.

File: content/en/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+This pod configuration file describes a pod that has a `requiredDuringSchedulingIgnoredDuringExecution` node affinity,`disktype: ssd`.
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md]:
```suggestion
This manifest describes a Pod that has a `requiredDuringSchedulingIgnoredDuringExecution` node affinity,`disktype: ssd`.
```

File: content/en/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+    `disktype=ssd` label.
+
+## Assign pod to node using required node affinity
+
+This pod configuration file describes a pod that has a `requiredDuringSchedulingIgnoredDuringExecution` node affinity,`disktype: ssd`. 
+This means that the pod will get scheduled only on a node that has a `disktype=ssd` label. 
+
+{{< codenew file="pods/pod-nginx-required-affinity.yaml" >}}
+
+1. Use the configuration file to create a pod that will get scheduled on your
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md]:
```suggestion
1. Apply the manifest to create a pod that is scheduled onto your
```

File: content/en/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+1. Verify that the pod is running on your chosen node:
+
+    ```shell
+    kubectl get pods --output=wide
+    ```
+
+    The output is similar to this:
+    
+    ```shell
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md]: This is sample output, not shell code.

File: content/en/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md]: This is sample output, not shell code

File: content/en/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md (hunk 1 above)
Comment (rajeshdeshpande02) [reply content/en/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md]: @sftim , I am not sure hot to add `min-kubernetes-server-version` Can you please point me to exisitng page where this is used. Or can I remove `{{< version-check >}} if it is not required.`

**Comment by rajeshdeshpande02** [comment]
@sftim  Can you please review this. I have implemented the review comments

**Comment by sftim** [comment]
Should be able to look within the next 48 hours

File: content/en/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md]: Have a look at line 3 of https://raw.githubusercontent.com/kubernetes/website/23a28d6dd288293d6d2970a9381f48753b7969fe/content/en/docs/tasks/configure-pod-container/share-process-namespace.md#L3

**Comment by jimangel** [comment]
It appears this PR might need the CI kicked (more info: https://github.com/kubernetes/website/pull/19692#issuecomment-601656207)

**Comment by rajeshdeshpande02** [comment]
Hello @jimangel , Thanks for your review. Have kicked CI with some style issues fixed. Please have a loot at it: 
New page added is :
https://deploy-preview-19265--kubernetes-io-master-staging.netlify.com/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity/

**Comment by sftim** [comment]
/hold
pending v1.18 Kubernetes release

**Comment by VineethReddy02** [comment]
/hold cancel
v1.18 release is out.

**Pull request merged**